packages/flex-plugins-api-toolkit/src/scripts/createConfiguration.ts

```diff
@@ -77,7 +105,8 @@ export default function createConfiguration(
         })
         .filter(Boolean),
     );
-    const list: string[] = option.addPlugins;
+    const list: string[] = [];
+    // = option.addPlugins;
```

Comment:
revert?

Reply:
Na gotta delete this actually. No longer used. The implementation has changed.